README.md

```diff
@@ -1 +1,50 @@
-# DSCI_522_group_31
\ No newline at end of file
+# Online Purchasing Intention Predictor
+
+- author: Yazan Saleh
```

Comment:
Although you are the author of this document, the README file is for the whole project, I would think this is our group project, all members should be contributors.

Reply:
I did list everyone under contributors, or did you mean to say all members should be authors as well?

Reply:
I mean there should be no author :D

Reply:
Or should everyone be listed as co-authors?

Reply:
I think you must've meant to say all members should be authors, which i agree with. This makes it seem to an outsider that I am the only one working on the project which isn't the case. I initially misinterpreted it as author of the README file.